documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="collected-logs-cr-info_{context}"]
+= Collected logs and custom resource information
+
+You can download logs and custom resource (CR) `yaml` files for the following targets by using the {project-short} web console or the command line interface (CLI):
+
+* Migration plan: Web console or CLI.
+* Virtual machine: Web console or CLI.
+* Namespace: CLI only.
++
+[NOTE]
+====
+If you specify a non-existent target, the `must-gather` archive file is empty.
+
+Empty or excluded log files are not included in the `must-gather` archive file.
+====
+
+The `must-gather` tool collects the following logs and CR files in an archive file:
+
+* Logs:
+** `importer` pod: Data volume conversion log.
+** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM.
+** `virt-launcher` pod: VM launcher log. When a migrated VM is powered on, the `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
+** `forklift-controller` pod: Filtered for the migration plan, virtual machine, or namespace.
+** `forklift-must-gather-api` pod: Filtered for the migration plan, virtual machine, or namespace. This log is not available if you run a targeted `must-gather` command from the CLI.
+
```

Comment:
Please remove this line: "This log is not available if you run a targeted `must-gather` command from the CLI."
It is not true for all cases
Thanks